src/EnergyPlus/HVACUnitarySystem.cc

```diff
@@ -7127,12 +7681,23 @@ namespace HVACUnitarySystem {
         OnOffAirFlowRatio = 1.0;
         CoilCoolHeatRat = 1.0;
 
```

Comment:
Simple fix for set point based control using ATMixer was to add these calls before and after the coils/fan are simulated in the set point calc routine.